pkg/network/protocols/http/testutil/testutil.go

```diff
@@ -51,6 +81,14 @@ func HTTPServer(t *testing.T, addr string, options Options) func() {
 		io.Copy(w, req.Body)
 	}
 
+	/* Save and recover TCP timestamp option */
+	oldTCPTS := false
+	if options.EnableTCPTimestamp != nil {
+		oldTCPTS = isNetIPV4TCPTimestampEnabled(t)
+		setNetIPV4TCPTimestamp(t, *options.EnableTCPTimestamp)
+		t.Cleanup(func() { setNetIPV4TCPTimestamp(t, oldTCPTS) })
+	}
```

Comment:
declaration in line 85 is redundant as the variable is being used only inside the c
```suggestion
	if options.EnableTCPTimestamp != nil {
		oldTCPTS := isNetIPV4TCPTimestampEnabled(t)
		setNetIPV4TCPTimestamp(t, *options.EnableTCPTimestamp)
		t.Cleanup(func() { setNetIPV4TCPTimestamp(t, oldTCPTS) })
	}
```